web-animations-1/Overview.bs

```diff
@@ -628,6 +628,21 @@ a document's load event.
 A <a>timeline</a> is considered to be <dfn lt="inactive timeline">inactive</dfn>
 when its <a>time value</a> is <a>unresolved</a>.
 
+A [=timeline=] may be in one of four possible <dfn>phase</dfn>s:
+
+* <dfn lt="inactive phase">inactive</dfn>
+* <dfn lt="before phase">before</dfn>
+* <dfn lt="active phase">active</dfn>
+* <dfn lt="after phase">after</dfn>
```

Comment:
This should probably be integrated with the previous paragraph (e.g. introduce the phases and then say that when it's inactive there's no meaningful time value).